### PR TITLE
fix(textarea): label color and focus highlight

### PR DESCRIPTION
**Pull request by sean-perkins**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
  - Some docs updates need to be made in the `ionic-docs` repo, in a separate PR. See the [contributing guide](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#modifying-documentation) for details.
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

The modern textarea rendering does not account for a focus highlight or changing the floating label color.

Issue URL: Internal

## What is the new behavior?

- Textarea will display a focus highlight when the control is focused
- Textarea label will change to the focus highlight color when active
- Textarea active caret color will change to the focus highlight color when active/focused

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by sean-perkins** [comment]
I'm skipping a flaky test here: https://github.com/ionic-team/ionic-framework/pull/26664/commits/a842a78b378f9030e6c3809efb1c9d7f5486fc30.

I can create a new PR/build that adds a ticket to resolving it. It took 3-4x runs for screenshots to generate and 3x runs after commenting out this test for it to pass (all from flaky tests).

**Comment by liamdebeasi** [comment]
Oddly enough I just fixed that test here https://github.com/ionic-team/ionic-framework/pull/26667 😂

File: core/src/components/textarea/textarea.scss [hunk 1]
```diff
@@ -149,6 +151,7 @@
   outline: none;
 
   background: transparent;
+  color: inherit;
```
Comment (liamdebeasi) [inline core/src/components/textarea/textarea.scss:154]: What does this do?

**Review by liamdebeasi — APPROVED** [review]

File: core/src/components/textarea/textarea.scss (hunk 1 above)
Comment (sean-perkins) [reply core/src/components/textarea/textarea.scss:154]: Good question - I think I was adding it during testing and didn't clean it up after. I'll test more after my early meetings and confirm. 

It looks like the cursor color also needs to be updated here as well.

**Pull request merged**